src/Symfony/Component/Form/Event/PostSetDataEvent.php

```diff
@@ -20,4 +21,11 @@
  */
 final class PostSetDataEvent extends FormEvent
 {
+    /**
+     * @deprecated since Symfony 6.4, it will throw an exception in 7.0.
+     */
+    public function setData(mixed $data): void
+    {
+        // throw new BadMethodCallException('Form data cannot be changed during "form.post_set_data", you should use "form.pre_set_data" instead.');
```

Comment:
missing runtime deprecation, isn't it? same in the other class

Reply:
Thanks, done.